blob/service.go

```diff
@@ -86,6 +93,67 @@ func DefaultSubmitOptions() *SubmitOptions {
 	}
 }
 
+func (s *Service) Start(context.Context) error {
+	s.ctx, s.cancel = context.WithCancel(context.Background())
+	return nil
+}
+
+func (s *Service) Stop(context.Context) error {
+	s.cancel()
+	return nil
+}
```

Comment:
Would be great to have graceful subscribtions shutdown logic here. Stop() should wait for all all ongiong subscriptions to be termintaed before exiting. Could be implemented as separate PR if you don't have time for it right now.

Reply:
could you check the latest commit to verify?